### PR TITLE
Update Add Command and Edit Command to support Skills and InterviewScore

### LINKED ISSUE #101: Add parameter s/SKILL for add command

### PR DESCRIPTION
**Pull request by kushaldas27**
Closes #101 and #102

### CODE REVIEW

**Comment by bigismols** [comment]
LGTM!

**Review by LemonDrew — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -137,27 +146,31 @@ public static class EditPersonDescriptor {
...
          */
         public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setName(toCopy.name);
             setJob(toCopy.job);
             setPhone(toCopy.phone);
             setEmail(toCopy.email);
+            setSkills(toCopy.skills);
+            setInterviewScore(toCopy.interviewScore);
             setTags(toCopy.tags);
         }
```
Comment (LemonDrew) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:167]: Good use of defensive code!

**Review by LemonDrew — APPROVED** [review]

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -44,11 +49,14 @@ public AddCommand parse(String args) throws ParseException {
         Job job = ParserUtil.parseJob(argMultimap.getValue(PREFIX_JOB).get());
```
Comment (SQ77) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:49]: For line 47, should add in `PREFIX_INTERVIEW_SCORE` for `argMultimap.verifyNoDuplicatePrefixesFor`?

**Pull request closed without merging**